workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.c2857a4.css

```diff
@@ -91,13 +91,14 @@
 #sidebar #spec-sidebar {
   padding-bottom: 100px;
 }
-#sidebar .sidebar-list li.list-title {
+#sidebar .sidebar-list .spec.list-title {
   font-weight: bold;
   color: #262626;
   font-size: 16px;
-}
-#sidebar .sidebar-list li.spec.list-title {
-  padding-left: 1.8rem;
+  padding: 0.3rem 0.2rem 0.3rem 1.8rem;
+  font-family: "Roboto", "Helvetica Neue", Arial, sans-serif;
```

Comment:
Is this necessary for a specific issue? Wondering if it could inadvertently override certain custom CSS rules.

Reply:
I believe that's the only way to override the `font-family` since the `h2` tag is styled [here](https://github.com/Kong/kong-portal-templates/blob/476c66ec892d29f656a0a5f88310ff3cf684701c/workspaces/default/themes/base/partials/theme/style.html#L41), so the `font-family` cannot be inherited from `#sidebar { ... }`.

We change this because in FTI-5334, the chrome extension `axe DevTools - Web Accessibility Testing` complains about the DOM structure 
```
3- <ul> and <ol> must only directly contain <li>, <script> or <template> elements:
to fix this, remove role=none from <li role="none" class="spec list-title">Resources</li>
```

Although just removing the `role="none"` can suppress the warning, I still think using `<ul>` to describe a nav bar is not enough for the semantics, so I changed the DOM structure by replacing `<ul>` to `<nav>`, also `<h2>`/`<div>`/`<ul>` to its children.

Do you know whether it's common for our customers to use CSS custom rules? I can change it back to original DOM structure if it's necessary

Reply:
That makes sense.

It is common for customers to use custom CSS to get the look and feel they need for their brand, but it is difficult to say which parts and what selectors they might use. We've been trying to balance making potential breaking changes to the DOM structure vs. improving the accessibility. It is a difficult tradeoff that won't satisfy every customer, since some want more accessible HTML and some would prefer no changes since they could break their customizations.

I agree about the semantic improvements being valuable to the accessibility. Just wanted to double-check about the reasoning for adding font-family since that one seems more likely to be caught in the middle of a visual change for customers who want to upgrade their templates.